packages/itwinui-css/backstop/tests/surface.html

```diff
@@ -27,16 +29,17 @@
         margin: calc(var(--iui-size-l) * -1);
         padding: var(--iui-size-l);
       }
-      .demo-surface-title-bar {
-        display: flex;
-        align-items: center;
-        height: calc(var(--iui-size-s) * 4);
-        border-bottom: 1px solid var(--iui-color-border);
-        padding: 0 var(--iui-size-s);
-      }
-      .demo-surface-content {
-        padding: var(--iui-size-s);
+      #demo-default {
+        display: grid;
+        grid-template-columns: repeat(
+          auto-fill,
+          minmax(Min(100%, 512px), 1fr)
+        );
+        gap: var(--iui-size-l);
+
+        --iui-surface-content-height: 246px;
```

Comment:
Setting the height of all surfaces nested within.  Try removing this to see all surfaces stretch to fit all content rather than using overflow / scroll.